integration-requirements.txt

```diff
@@ -1,8 +1,8 @@
-pytest-asyncio==0.21.1
+macaroonbakery==1.3.2
```

Comment:
Add a comment with a todo to remove this when it is fixed. Relevant PR: https://github.com/go-macaroon-bakery/py-macaroon-bakery/pull/95/

Reply:
thanks. added the comment.